code/modules/food_and_drinks/machinery/smartfridge.dm

```diff
@@ -19,6 +19,8 @@
 	var/list/initial_contents
 	/// If the machine shows an approximate number of its contents on its sprite
 	var/visible_contents = TRUE
+	/// Is this smartfridge going to have a glowing screen? (Drying Racks are not)
+	var/has_emissive
```

Comment:
Wait, is this meant to default to `TRUE`? It is not true anywhere

Reply:
If it's meant to be like this it should default to `FALSE` instead of null

Reply:
Uhhhhhh one second

Reply:
Fixed, I just forgot to set it True by default 💀